addon/components/sl-calendar-month.js

```diff
@@ -81,11 +73,14 @@ export default Ember.Component.extend({
     /**
      * The short string name of the represented month
      *
-     * @function shortName
-     * @returns  {Ember.String}
+     * @function
+     * @observes month
```

Comment:
The @observes tag is unnecessary due to our planned future use of a JSDoc plugin that derives this information automatically.